src/CompanyService.Validation/CreatePositionRequestValidator.cs

```diff
@@ -12,9 +12,12 @@ public CreatePositionRequestValidator()
                 .NotEmpty()
                 .MaximumLength(80);
 
-            RuleFor(position => position.Description)
-                .NotEmpty()
-                .MaximumLength(350);
+            When(position => !string.IsNullOrEmpty(position.Description), () =>
```

Comment:
only null, or it will be strange, can't be empty when not empty, and can be empty when empty)

Reply:
Fixed